Assets/MixedRealityToolkit/_Core/Devices/MixedReality/WMRDevice.cs

```diff
@@ -88,47 +88,99 @@ private IMixedRealityInputSource GetOrAddWindowsMixedRealityController(Interacti
 
             bool controllerTracked = ReadMixedRealityControllerValues(ref interactionSourceState);
 
-            // Define Controller capabilities
-            var controllerCapabilities = new List<InputType>();
+            var DetectedController = new WindowsMixedRealityController()
+            {
+                SourceId = interactionSourceState.source.id,
+                Handedness = interactionSourceState.source.handedness == InteractionSourceHandedness.Left ? Handedness.Left : Handedness.Right,
```

Comment:
How do we handle hands or controllers that done have any handedness, like the clicker?

Reply:
Well, we have Handedness.None for those, or we can re-think.

I'm also considering restructuring the relationship between "device" and "Controllers" as I'm starting to feel we need a separate "detection" logic for controllers.


Something less that the forced approach of VRTK and definitely separate to the "all in one" approach in current HoloToolkit.

But want to formalise these first controllers, first.